spec/system/consumer/checkout/payment_spec.rb

```diff
@@ -331,6 +331,12 @@
           end
         end
       end
+
+      describe "show page title as Checkout Payment - Open Food Network" do
+        it "should display title as Checkout Payment - Open Food Network" do
+          page.has_title? "Checkout Payment - Open Food Network"
+        end
+      end
```

Comment:
System specs are relatively expensive / slow to run. So when possible, it's better to add the test code to an existing `it` block.